### ISSUE TITLE
ACF Extended asks for authentication when installing

### ISSUE THREAD

**Issue by izzyzzi** [junaidbhura/composer-wp-pro-plugins#63]
Hi,

I'm installing ACF Pro, ACF Extended Pro, and Gravity Forms using your awesome composer package.

The other 2 have worked flawlessly, but when composer is installing ACF Extended, it prompts me to input a username and password

My composer.json has the repos set up:
```javascript
"repositories": [
        {
            "type": "package",
            "package": {
                "name": "junaidbhura/acf-extended-pro",
                "version": "0.8.9.3",
                "type": "wordpress-plugin",
                "dist": {
                    "type": "zip",
                    "url": "https://www.acf-extended.com/"
                },
                "require": {
                    "junaidbhura/composer-wp-pro-plugins": "*"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "junaidbhura/advanced-custom-fields-pro",
                "version": "6.1.4",
                "type": "wordpress-plugin",
                "dist": {
                    "type": "zip",
                    "url": "https://www.advancedcustomfields.com/"
                },
                "require": {
                    "junaidbhura/composer-wp-pro-plugins": "*"
                }
            }
        },
        {
            "type": "package",
            "package": {
                "name": "junaidbhura/gravityforms",
                "version": "2.7.4",
                "type": "wordpress-plugin",
                "dist": {
                    "type": "zip",
                    "url": "https://www.gravityforms.com/"
                },
                "require": {
                    "junaidbhura/composer-wp-pro-plugins": "*"
                }
            }
        }
    ]
``` 

and the packages added to require:

```javascript
"require": {
        "junaidbhura/composer-wp-pro-plugins": "^1.8",
        "junaidbhura/acf-extended-pro": "*",
        "junaidbhura/advanced-custom-fields-pro": "*",
        "junaidbhura/gravityforms": "*"
    },
``` 

I input the ACF Extended Pro key and the URL into the .env file (as well as the other plugins keys).

As I mentioned ACF Pro and Gravity both work flawlessly , Only ACF Extended prompts me. Is this normal behaviour for ACF Extended using this tool? Any guidance would be very appreciated, and thanks for your awesome work!

**Comment by mcaskill** [comment]
It  requests a username/password because ACF Extended Pro uses Easy Digital Downloads (same for Ninja Forms, Polylang, PublishPress, WP All Import).

You will need to add the username/password of your ACF Extended account to Composer's `auth.json`:

```
{
  "http-basic": {
      "acf-extended.com": {
          "username": "...",
          "password": "..."
      }
  }
}
```

**Comment by junaidbhura** [comment]
@mcaskill We should probably update the Readme to include this info?

**Comment by mcaskill** [comment]
> We should probably update the Readme to include this info?

Good idea, a general notice about this occurrence, and which plugins are affected, would be practical.

**Comment by izzyzzi** [comment]
> It requests a username/password because ACF Extended Pro uses Easy Digital Downloads (same for Ninja Forms, Polylang, PublishPress, WP All Import).
> 
> You will need to add the username/password of your ACF Extended account to Composer's `auth.json`:
> 
> ```
> {
>   "http-basic": {
>       "acf-extended.com": {
>           "username": "...",
>           "password": "..."
>       }
>   }
> }
> ```

Thanks all, that has worked! 
Not sure whether I should close the ticket or whether you wanted to leave it open? And thanks again for such a useful tool, and your speedy responses. Love it.

**Comment by influactivecom** [comment]
Hello,

It's not working for me :
```
{
  "http-basic": {
    "connect.advancedcustomfields.com": {
      "username": "***",
      "password": "http://***.test"
    },
    "acf-extended.com": {
      "username": "email",
      "password": "password"
    }
  }
}
```

**Comment by influactivecom** [comment]
I tried with "www.acf-extended.com" but the download remains at 0%

**Comment by mcaskill** [comment]
@influactivecom Are you still having issues with ACF Extended Pro?

It could be an SSL certificate problem which requires updating the `wp-includes/certificates/ca-bundle.crt`. See this [comment in #64](https://github.com/junaidbhura/composer-wp-pro-plugins/issues/64#issuecomment-1559061281). If you copy over the changes from its linked pull request and try again, we might be able to see a more relevant error.

**Comment by yaroslav-borodii** [comment]
> I tried with "www.acf-extended.com" but the download remains at 0%

1. Check if you have added your production/staging/local **domain** to your **ACF Extend Account**.
2. The `Auth.json` configuration domain must contain `www`.
```
{
  "http-basic": {
      "www.acf-extended.com": {
          "username": "...",
          "password": "..."
      }
  }
}
```
3. If this does not help, check on errors with the `-vvv' optional command.

```
composer update -vvv
```

![untitled (2)](https://github.com/junaidbhura/composer-wp-pro-plugins/assets/64497493/df1c1e6a-b8f3-4c9c-823a-446286171502)

![untitled (3)](https://github.com/junaidbhura/composer-wp-pro-plugins/assets/64497493/3fe00cd1-e488-4d29-a5bf-fee1a7bec976)
